apps/chat-e2e/src/testData/expectedConstants.ts

```diff
@@ -280,6 +280,7 @@ export enum ModelIds {
   STABLE_DIFFUSION = 'stability.stable-diffusion-xl',
   IMAGE_GENERATION_005 = 'imagegeneration@005',
   GEMINI_PRO_1_5 = 'gemini-1.5-pro-preview-0409',
+  GEMINI_FLASH_1_5 = 'gemini-1.5-flash-001',
```

Comment:
We don't have such model on Review-environment

Reply:
I've just added it:

https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/merge_requests/100/diffs